run-build.ps1

```diff
@@ -106,13 +106,13 @@ if($Architecture.StartsWith("arm", [StringComparison]::OrdinalIgnoreCase))
 Write-Output "$dotnetInstallPath -version ""$dotnetCliVersion"" -InstallDir $env:DOTNET_INSTALL_DIR -Architecture ""$InstallArchitecture"""
 Invoke-Expression "$dotnetInstallPath -version ""$dotnetCliVersion"" -InstallDir $env:DOTNET_INSTALL_DIR -Architecture ""$InstallArchitecture"""
 
-Invoke-Expression "$dotnetInstallPath -Version ""1.1.2"" -SharedRuntime -InstallDir $env:DOTNET_INSTALL_DIR -Architecture ""$Architecture"""
-
 if ($LastExitCode -ne 0)
```

Comment:
Should this check now be removed or were we missing corresponding checks for the previous `Invoke-Expression`?

Reply:
I still want the check to happen for the regular SDK. I just don't want it to fail when installing the backwards compatible runtime, as it won't be available on all OSes.

Reply:
Cool, just wasn't sure if we still needed it since no check was done for the preceding command before.